backend/api/go_client/BUILD.bazel

```diff
@@ -0,0 +1,39 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "error.pb.go",
```

Comment:
can this be wildcard *.pb.go *.pb.gw.go? otherwise this need to manually added in the future i guess?